Docs: adding Questions and Configs (#3975)
minvws/nl-kat-coordination: docs/source/manual/user-manual.rst

Reviewer: Disallowed CSP hostnames:
De omschrijving is wat algemeen. Als gebruiker begrijp ik toch niet precies de actie die wordt verwacht, welke syntax wordt vereist en welk risico vermijden we hiermee. Worden hier scan targets vermeden? En krijg je daarmee een blinde vlek in de scan? Is het mogelijk naar een voorbeeld te verwijzen?

OOIs in headers
Wat vul je hier precies in met welke syntax leidend tot welk resultaat?

OOIs in headers
Ook niet helemaal duidelijk voor mij als eenvoudige gebruiker (zie eerdere opm. voor strekking)

typo's:
- ... "system. FQThe new users" ... : Is FQ hier bewust toegevoegd?
- ... "against your applied policy.." ... >> ... "against your applied policy." ...
- ... "a form in the web interface.." ... >> ... "a form in the web interface." ...
- ... "to set an policy" ... >> ... "to set a policy" ...

Author: Very valid feedback! I'll make sure to update the text accordingly :pray:

**Limitations**
It only checks for parameters in the URL, headers are ignored. There is currently no way to specify the parameters per hostname.

**Examples**
Provide the URL parameter that is to be ignored, separate each parameter by a comma. URL parameters can be found in the URL bar after the `?` sign. They are the values after the `&` and before the `=` signs.

The URL is: `https://example.com/index.php?session_id=12346&search=meow <dontredirect>`_ . This URL has two parameters:`session_id` and `search`, which both have a value (`123456` and `meow`). The parameter and value for `session_id=123456` are expected to clutter the Objects list, as for each new session the value `123456` is different. The next time you visit the website this might be `session_id=43215` and the next time `session_id=958367`. You can reduce the object clutter by adding the parameter `session_id` to the list as this will be the same for each session. Adding the value `123456` will not work, as this will be different each time.

Port mapping
------------

**Why is this question introduced?**

> Maps open ports into specific categories based on the services in your environment. You can specify common TCP and UDP ports which will not trigger a finding (such as e-mail server ports). By adding ports to the database (db) and system administrator (sa) lists you tell OpenKAT which ports are likely used in your network. Depending on your port mapping OpenKAT will show different findings if the port is detected. The `aggregate_findings` functionality allows you to group findings on the IP-address instead of as separate ports.

Reviewer 2: ```suggestion
Maps open ports into specific categories based on the services in your environment. You can specify common TCP and UDP ports which will not trigger a finding (such as e-mail server ports). By adding ports to the database (db) and system administrator (sa) lists you tell OpenKAT which ports are likely used in your network. Depending on your port mapping OpenKAT will show different findings if the port is detected. The `aggregate_findings` functionality allows you to group findings by IP address rather than treating them as separate ports.
```
